src/components/AppSlider/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import Slider from 'rc-slider';
+
+export const appSliderDataTestIds = {
+  base: 'app-slider__base',
```

Comment:
As this is the root element, do we need the `__base`?

Reply:
@hoangmirs yeah, I apply `base` test id to every root of an object 😄 
![image](https://github.com/manh-t/react-survey/assets/60863885/5b9a4ad7-f3f1-489e-a4a7-e23402fedb28)